src/cf-core/usage.md

```diff
@@ -496,10 +529,6 @@ To create a 4:3 flexible video player, add the `__4_3` modifier to the container
 </div>
```

Comment:
This section is a little warped in the preview but I can't find anything wrong. Can you take a look, and if this is an issue with my preview, we can take a look after we merge. 

![image](https://cloud.githubusercontent.com/assets/1860176/17025908/0c345e3a-4f2c-11e6-8505-29e8eafa7e96.png)